### PR TITLE
Adds manual tablet mode setting

### LINKED ISSUE #6044: In Split Screen maximize video
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

When in split screen mode, the video should be maximized to fill the area, the way the official Youtube app does it.
Screenshot_20210410-231531

At least, when the full screen button is pressed, this should be the result. Currently, the full screen button does nothing in split screen mode.

#### Is your feature request related to a problem? Please describe it

When I use NewPipe in split screen mode, the app tries to squeeze the full view into the tiny split screen view, instead of displaying only the video. This makes it very tiny.
Screenshot_20210410-231448

#### How will you/everyone benefit from this feature?

This feature would allow multitasking while watching videos. Currently, I am stuck with a tiny video and half the screen wasted.

### PR DESCRIPTION
**Pull request by Dakkaron**
#### What is it?
- [ ] Bugfix (user facing)
- [x] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Adds a setting to manually force tablet mode to be on or off under Appearance settings
- Works as a work-around for tablet-mode related issues like #5967
- Also helps all users who increase the minimum width to fit more stuff on their screen, but prefer the better-fitting phone mode for NewPipe

#### Fixes the following issue(s)

- Fixes #6044 

#### APK testing 

https://github.com/TeamNewPipe/NewPipe/suites/2474591406/artifacts/53233850

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -714,4 +714,7 @@
     <string name="night_theme_summary">Select your favorite night theme — %s</string>
     <string name="select_night_theme_toast">You can select your favorite night theme below</string>
     <string name="download_has_started">Download has started</string>
+    <string name="tablet_mode_title">Tablet Mode</string>
+    <string name="on">On</string>
+    <string name="off">Off</string>
```
Comment (mhmdanas) [inline app/src/main/res/values/strings.xml:719]: These are translatable, which we almost definitely don't want them to be.

**Review by mhmdanas — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (mhmdanas) [reply app/src/main/res/values/strings.xml:719]: Actually, ignore that. I thought you were using them in the Java code, but it turns out you're not.

**Comment by Dakkaron** [comment]
Are you sure? These are user-facing strings. For the non-userfacing key IDs (which should not be translatable and are marked as non-translatable) there are `@string/tablet_mode_auto_key`, `@string/tablet_mode_on_key` and `@string/tablet_mode_off_key`.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Dakkaron) [reply app/src/main/res/values/strings.xml:719]: Are you sure? These are user-facing strings. For the non-userfacing key IDs (which should not be translatable and are marked as non-translatable) there are `@string/tablet_mode_auto_key`, `@string/tablet_mode_on_key` and `@string/tablet_mode_off_key`.

Comment (mhmdanas) [reply app/src/main/res/values/strings.xml:719]: No, I was mistaken. Ignore my comment.

Comment (Dakkaron) [reply app/src/main/res/values/strings.xml:719]: Sorry, Github doesn't auto-update, so I didn't see your second comment before posting. So please disregard my comment as well.

**Comment by Dakkaron** [comment]
Is there anything else I need to do to get this PR merged?

**Comment by mhmdanas** [comment]
@Dakkaron someone from the team who is actually familiar with Android apps would have to review it and approve it. I'll request reviews from a couple people now.

File: app/src/main/java/org/schabi/newpipe/util/DeviceUtils.java [hunk 2]
```diff
@@ -55,6 +57,16 @@ public static boolean isTv(final Context context) {
     }
 
     public static boolean isTablet(@NonNull final Context context) {
+        final String tabletModeKey = context.getString(R.string.tablet_mode_key);
+        final String tabletModeSetting = PreferenceManager.getDefaultSharedPreferences(context)
+                                                    .getString(tabletModeKey, "");
```
Comment (TiA4f8R) [inline app/src/main/java/org/schabi/newpipe/util/DeviceUtils.java:62]: I am not an Android developer or not very familiar with Android apps, so I cannot review the code itself but I think if the indentation isn't correct on these lines. Here is what I got when I paste the code in Android Studio:
```suggestion

        final String tabletModeSetting = PreferenceManager.getDefaultSharedPreferences(context)
                .getString(tabletModeKey, "");
```

Comment (Dakkaron) [reply app/src/main/java/org/schabi/newpipe/util/DeviceUtils.java:62]: This kind of indentation is pretty much preference. There are coding styles that say to indent the one way or the other. From what I could find, the contribution guidelines only said to stick with checkstyle, which hasn't flagged this.

But either way, I don't really care. So if that's the preferred way, I can change it like that.

**Review by mhmdanas — DISMISSED** [review]
LGTM. Waiting on other reviewer(s).

**Review by Stypox — APPROVED** [review]
Great, thank you! I added a comment before merging, just so that auto mode is clearer in code.

**Pull request merged**